src/composition/watchers/watcher/introspection.rs

```diff
@@ -50,8 +50,14 @@ impl SubgraphIntrospection {
             .skip(1)
             .filter_map(|change| async move {
                 match change {
-                    Ok(subgraph) => Some(subgraph),
+                    Ok(subgraph) => {
+                        eprintln!("Connectivity restored for subgraph \"{}\".\n", subgraph.name());
```

Comment:
The specs call for an event to be fired, and a central messaging module to handle this communication. I don't see an example of that, and suspect it might not have made the final cut. Let me know if there is a better way to emit messages to the user that is more in line with current design approach.